source/System.cpp

```diff
@@ -276,7 +277,19 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 					}
 			}
 			else
-				fleets.emplace_back(fleet, child.Value(valueIndex + 1));
+			{
+				fleets.emplace_back(fleet);
+				LimitedEvents<Fleet> &fleet = fleets.back();
+
+				if(child.Size() > valueIndex + 1)
+					fleet.Period() = child.Value(valueIndex + 1);
```

Comment:
I've never seen anything like this assignment...  Is this possible only because of 
https://github.com/endless-sky/endless-sky/blob/d4fd20b3bb7040b74cbf27c698c41cc01f07c675/source/RandomEvent.h#L32
??